### PR TITLE
Implemented ReflectionMethod::getClosure()

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/Reflection/ReflectionMethod.php [hunk 1]
```diff
@@ -294,4 +299,71 @@ public function getImplementingClass() : ReflectionClass
...
+        };
+    }
+
+    /**
+     * @throws ClassDoesNotExist
+     */
+    private function assertClassExist(string $className) : void
+    {
+        if ( ! \class_exists($className, false)) {
+            throw new ClassDoesNotExist('Method cannot be used as the class is not loaded');
```
Comment (Ocramius) [inline src/Reflection/ReflectionMethod.php]: Please add the class name to the exception

**Comment by Ocramius** [comment]
Ah, and a rebase will be needed, sorry :-\

**Comment by kukulich** [comment]
@Ocramius No problem, done :)

File: src/Reflection/ReflectionMethod.php (hunk 1 above)
Comment (kukulich) [reply src/Reflection/ReflectionMethod.php]: Done.

**Comment by Ocramius** [comment]
Good to go! :+1:

**Pull request merged**